specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -2453,6 +3788,24 @@
         "modelAsString": true
       }
     },
+    "ManagedOnlineEndpointDeploymentResourceProperties": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EndpointDeploymentResourceProperties"
+        }
+      ],
+      "x-ms-discriminator-value": "managedOnlineEndpoint"
```

Comment:
>managedOnlineEndpoint [](http://example.com/codeflow?start=35&length=21)

This discriminator value is duplicate of the one below, but with different schema. It needs to be distinguishable.

Reply:
they are discriminator for two different base object.